compiler/esy.json

```diff
@@ -25,14 +25,14 @@
     "copy:exe": "cp -f #{$GRAINC_BIN_PATH} #{$GRAINDOC_BIN_PATH} #{$GRAINFORMAT_BIN_PATH} #{$GRAINLSP_BIN_PATH} #{$CLI_BIN_DIR}",
     "copy:js": "cp -f #{$GRAINC_JS_PATH} #{$GRAINDOC_JS_PATH} #{$GRAINFORMAT_JS_PATH} #{$GRAINLSP_JS_PATH}  #{$CLI_BIN_DIR}",
     "clean": "rm -rf #{self.root}/_esy",
-    "build:js": "esy b dune build @js --no-buffer #{os == 'windows' ? '-j 2' : ''}",
+    "build:js": "esy b dune build @js --no-buffer -j 2",
```

Comment:
I lowered the number of builders as we were running into out-of-memory issues when building on Ubuntu, likely because binaryen has grown.

I decided to lower on all platforms because I noticed that we were actually building faster with fewer workers, even on macOS where there was enough RAM available, likely do to less memory pressure, causing less swap usage. I noticed similar behaviour when building locally previously. I saw macos build times go from ~24minutes to ~15minutes


I went with 2 as it's what we were using on Windows previously; however, it may make sense to play with this number a little more in a separate pr for optimum performance.